ops/docker/ci-builder/Dockerfile

```diff
@@ -10,7 +10,7 @@ SHELL ["/bin/bash", "-c"]
 
 ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update && \
-  apt-get install -y curl build-essential git clang lld curl
+  apt-get install -y curl build-essential git clang lld curl jq
```

Comment:
Why do we install twice?

Reply:
We could clean that up in a follow up PR, i'd like to get it published. No reason that i am aware of